pandas/core/aggregation.py

```diff
@@ -5,12 +5,99 @@
 
 from collections import defaultdict
 from functools import partial
-from typing import Any, Callable, DefaultDict, List, Sequence, Tuple, Union
+from typing import (
+    Any,
+    Callable,
+    DefaultDict,
+    Dict,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+)
+
+from pandas._typing import Label
 
 from pandas.core.dtypes.common import is_dict_like, is_list_like
 
+from pandas.core.base import SpecificationError
 import pandas.core.common as com
 from pandas.core.indexes.api import Index
+from pandas.core.series import FrameOrSeriesUnion, Series
+
+# types of `func` kwarg for DataFrame.aggregate and Series.aggregate
```

Comment:
great, we *may* want to think about putting these in _typing, but separate issue / PR (you can just try it if you want), but in a follown.

Reply:
ahh, okay! i deliberately moved it from _typing to here in commit 05921af since i thought this might be only used in this file.

I will do a PR for moving it to _typing.